drivers/regulator/regulator_fixed.c

```diff
@@ -93,14 +94,17 @@ static int regulator_fixed_init(const struct device *dev)
 			return -ENODEV;
 		}
 
-		int ret = gpio_pin_configure_dt(&cfg->enable, GPIO_OUTPUT);
+		ret = gpio_pin_configure_dt(&cfg->enable, GPIO_OUTPUT);
 
 		if (ret < 0) {
 			return ret;
 		}
 	}
 
-	return regulator_common_init(dev, false);
```

Comment:
It looks like this issue can also be resolved if you add a get_voltage handler to the fixed regulator driver. 

Moreover, I reviewed the usage of regulator-fixed nodes across the projects and found that most often developers haven't set min/max voltage in the device tree source, and they don't catch the issue you're facing. So, the simpler fix would be to remove the min/max properties from the DTS.